README.md

```diff
@@ -1,47 +1,54 @@
+This is the *flowR* extension for Visual Studio Code which allows you to retrieve program slices directly within your IDE. For more information on *flowR* and its capabilities, please check out [the main repository](https://github.com/Code-Inspect/flowr).
 
-Then, you can install it the same way as you would the `vsix` downloaded [from GitHub Releases](#from-github-releases).
+![A screenshot of the extension being used to reconstruct a slice](media/splash.png)
 
-## Using
+## Use
 
 **To use this extension, a working installation of [R](https://www.r-project.org/) is required**, and R needs to be included in your `PATH` environment variable. You may need to [do so manually](https://www.hanss.info/sebastian/post/rtools-path/) on Windows. (There are [plans](https://github.com/Code-Inspect/vscode-flowr/issues/5) to allow for the extension to install R automatically.)
 
 Although it is not required, we recommend using the [R extension](https://marketplace.visualstudio.com/items?itemName=REditorSupport.r) for Visual Studio Code along with this extension. For more information on R development in Visual Studio Code, you can also check out [this helpful article](https://code.visualstudio.com/docs/languages/r).
 
 ### Slicing
+
 You can generate a [slice](https://github.com/Code-Inspect/flowr/wiki/Terminology#program-slice) of the currently highlighted variable in any R code by using the "Slice for Cursor Position" command. All code that is not part of the generated slice will then be grayed out.
 
 To clear the slice highlighting, use the "Clear Slice Presentation" command.
 
 You can also reconstruct a piece of code based on the slice of a variable by using the "Slice for Cursor Position (Reconstruct)" command. The reconstructed code will be opened in a new file.
 
-## Developing
+## Installing
+
+### From Visual Studio Marketplace
+
+You can get the extension here: <https://marketplace.visualstudio.com/items?itemName=code-inspect.vscode-flowr>.
+
+### From GitHub Release
+
+You can find official releases of the extension in the [Releases](https://github.com/Code-Inspect/vscode-flowr/releases) section of the repository. Simply select the version you would like to download, open up the asset's section at the bottom, and download the `vscode-flowr-[version].vsix` contained in it.
+
+From Visual Studio Code, open the Extensions tab and click on the three dots in the top right to select "Install from VSIX..." Alternatively, you can use the Command Palette to select the option directly. Then, you can select the `vsix` file you downloaded to install it.
+
+### From Build Artifact
```

Comment:
These use titlecase now but the "Building and running from source" section still uses normal casing :(